File: Python/Multipleof3and5.py
# Problem 1: "Multiple of 3 and 5" of Project Euler

def multipleOf3And5(num):
  """ This fundction will check all the numbers from 1 to num 
  and return the sum of them. """

  # below list will hold all the numbers divisible by 3 and 5
  list = []

  #for loop will iterate over numbers from 1 to num 
  for i in range(1, num):
    #if and elif statement will check for divisibility by 3 and 5
    if(i % 3 == 0):
      list.append(i)
    elif(i % 5 ==  0):
      list.append(i)
  
  #sum of numbers of list
  sum_of_list = sum(list)
  return sum_of_list

print(multipleOf3And5(8456))

File: Javascript/Multipleof3and5.js
function multiplesOf3and5(number) {
    // Good luck!

    //Define an empty array to hold all the numbers
    var arr = [];

    // for loop will iterate over all the values from 1 to 'number'
    for(var i=1; i< number; i++){
      if(i % 3 == 0){
        arr.push(i);
      }
      else if(i % 5 == 0){
        arr.push(i)
      }
    }

    // below function will add all the values and return us the result
    const arrSum = function(arr){
      return arr.reduce(function(a,b){
        return a + b
      }, 0);
    }

    var result = arrSum(arr);
    return result;
}
  
  
  
multiplesOf3and5(1000);

console.log(multiplesOf3and5(1000));